docs/TLS.md

```diff
@@ -15,10 +14,29 @@ In addition, one has to pass the `tls-enabled yes` option to the redisraft modul
 --loadmodule ./redisraft.so <other options> tls-enabled yes
 ```
 
-RedisRaft is smart enough to figure out which files to use for the local cert/key/ca by querying the underlying Redis server.  However, if a user wants to use different files for the client aspects vs server aspects, one can specify RedisRaft module options as well
+## Configuring RedisRaft TLS support
 
-```asm
-tls-ca-cert
-tls-cert
-tls-key
-```
\ No newline at end of file
+RedisRaft can be configured by reusing the Redis server's TLS configuration, or by configuring the RedisRaft module directly.  In general, we expect people will configure the Redis server. 
+
+### Configuring RedisRaft via Redis.
```

Comment:
Why don't we use Redis config names?

Reply:
we can, just seemed we could be a little simpler.  not biggy.

Reply:
You mean with shorter names? I love short names but in this case, probably it's better to follow Redis config name. So, we don't maintain conversion table here and people don't have to look at it.